tools/README.md

```diff
@@ -0,0 +1 @@
+Download sokol-shdc from https://github.com/floooh/sokol-tools-bin/tree/bindings-cleanup
```

Comment:
Is the bindings-cleanup branch explicitly required now?

Reply:
it's the latest tag. I though it made sense to link to the specific version of the bin which generated the shaders.

checked with latest master https://github.com/floooh/sokol-tools-bin/tree/9b5a3e2b57fe9783ba4d1f3249059bc4720b592f and the output is the same